### ISSUE TITLE
ADA & Section 508 Compliance Issue - accessibility - navigation widget - attribute in the wrong place

### ISSUE THREAD

**Issue by Bovelett** [elementor/elementor#17582]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

When you have more than one navigation on a page, it's important for accessibility to be able to make a distinction between the menus. To give you an example: usually a header menu gets aria-label="primary" and footer gets aria-label="secondary". 
Any attribute for the navigation should be called **inside** the nav element. It's basic semantic HTML.

However, when you add an attribute to the Nav Menu widget, it's added to a div on a higher level. See screenshot.

![Screenshot 2022-01-25 at 23 22 08](https://user-images.githubusercontent.com/852140/151070552-c6bbf784-7b6b-4e4a-87b8-f4cebe96972c.png)

## Steps to reproduce

1. Add a Nav Menu widget
2. go to the Advanced Tab
3. add aria-label|primary

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Ubuntu 18.04 v5.6.49-89.0
	PHP Version: 7.4.27
	PHP Max Input Vars: 1000
	PHP Max Post Size: 16M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.9
	Site URL: https://annebovelett.dev
	Home URL: https://annebovelett.dev
	WP Multisite: No
	Max Upload Size: 10 MB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.4.2
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.36

== Active Plugins ==
	Code Snippets
		Version: 2.14.3
		Author: Code Snippets Pro

	Elementor
		Version: 3.5.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.5.2
		Author: Elementor.com

	Slim Maintenance Mode
		Version: 1.4
		Author: Johannes Ries

	Yoast SEO
		Version: 18.0
		Author: Team Yoast


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active
	Import Export Template Kit: Inactive
	Additional Custom Breakpoints: Inactive
	Hide native WordPress widgets from search results: Inactive
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive
	Landing Pages: Inactive
	Color Sampler: Inactive
	Favorite Widgets: Inactive
	Admin Top Bar: Inactive
	Form Submissions: Inactive
	Scroll Snap: Inactive


== Log ==
	
PHP: showing 2 of 2PHP: 2022-01-25 12:22:37 [notice X 1][/var/www/vhosts/annebovelett.dev/httpdocs/wp-content/plugins/elementor/core/common/modules/connect/module.php::230] Trying to get property 'email' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-01-25 19:58:34 [notice X 3][/var/www/vhosts/annebovelett.dev/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	

				
```
</details>

**Comment by shilo-ey** [comment]
Thanks Anne-Mieke! We will go over this and resolve this issue

**Comment by Bovelett** [comment]
Thank you! 🙏

**Comment by shilo-ey** [comment]
@Bovelett What are you wishing to achieve out of the box with the Nav Menu widget?

The Attributes resides on the wrapper of the element and not the element itself by design.

**Comment by Bovelett** [comment]
@shilo-ey I must apologize, as (partially) stand corrected! :-) I was taught to add attributes inside the semantic nav element. Turns out I was misinformed about the importance of the location of the attribute for the screen reader. It was not announcing the label I added to the Nav Menu widget. I tried again just now and feel stupid, I was using the wrong mode in the (Voice Over on Mac) screen reader. That'll teach me for working in the middle of the night!  

The redundancy of role=navigation (see yellow text in screenshot) inside a nav element still stands, but that has nothing to do with accessibility. This just gets flagged in an HTML audit.

Should I create a separate report for that?

**Comment by shilo-ey** [comment]
No, thank you. no need for an additional report.

We will verify the redundancy of the `role=navigation` added there and update.

Thanks

**Comment by avivu** [comment]
This issue has been resolved in Elementor Pro v3.8.0

Feel free to update

Thanks!

**Comment by Bovelett** [comment]
Hey @avivu Thank you! 🤗
